packages/graphql_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v5.1.1.beta.1
+
+## New Feature
+- added query onError & onComplete callbacks ([commit](https://github.com/zino-hofmann/graphql-flutter/commit/8342d245ececc6007744d963c9a5d7ff8ed83f13)). @fabis94 22-04-2022
+
+## New Feature
+- Decouple GraphQLClient Provider ([commit](https://github.com/zino-hofmann/graphql-flutter/commit/750ca051f25a89df0b2a69176feab1499426f9b9)). @budde377 19-04-2022
+
+
```

Comment:
Missed aggregation by section, bug reported in changelog.dart https://github.com/vincenzopalazzo/changelog.dart/issues/16